packages/sdk/explorer/src/block-explorer.ts

```diff
@@ -380,14 +380,11 @@ export class BlockExplorer {
       this.getContractMappingFromSourcifyAsProxy,
     ]
   ): Promise<ContractMapping | undefined> {
-    const mappings = await Promise.all(
```

Comment:
why remove the wrapping `Promise.all`?

Reply:
it actually used to not have promise.all and then there was some proposal which took forever to build and I tried to speed things up by running them in parallel. but now that i know more about how it all works i realized how wasteful that will be most of the time.